core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BpmBillingProjectLifecycle.scala

```diff
@@ -176,33 +184,39 @@ class BpmBillingProjectLifecycle(
     }
   }
 
-  private def cleanupLandingZone(landingZoneId: UUID,
-                                 projectName: RawlsBillingProjectName,
-                                 ctx: RawlsRequestContext
-  ): Future[Unit] =
-    (for {
-      // Note that this actually just starts a landing zone deletion job (and thus returns quickly).
-      // We are not attempting to ensure that the landing zone deletion completes successfully.
-      landingZoneResponse <- Future(blocking {
-        val response = workspaceManagerDAO.deleteLandingZone(landingZoneId, ctx)
-        logger.info(
-          s"Initiated deletion of landing zone $landingZoneId for BPM-backed billing project ${projectName.value}."
-        )
-        response
-      })
-
-      _ = Option(landingZoneResponse.getErrorReport).map { errorReport =>
-        logger.warn(
-          s"Unable to delete landing zone with ID $landingZoneId for BPM-backed " +
-            s"billing project ${projectName.value}: ${errorReport.getMessage}."
-        )
-      }
-    } yield ()).recover { case t: Throwable =>
+  /**
+    *  starts a landing zone deletion job
+    *  does not ensure that the landing zone deletion completes successfully.
+    */
+  private def cleanupLandingZone(
+    landingZoneId: UUID,
+    ctx: RawlsRequestContext
+  ): DeleteAzureLandingZoneResult = Try(workspaceManagerDAO.deleteLandingZone(landingZoneId, ctx)) match {
+    case Failure(e: ApiException) =>
+      val msg = s"Unable to delete landing zone: ${e.getMessage}"
+      throw RawlsExceptionWithErrorReport(StatusCode.int2StatusCode(e.getCode), msg)
+    case Failure(t) =>
       logger.warn(
-        s"Unable to delete landing zone with ID $landingZoneId for BPM-backed billing project ${projectName.value}.",
+        s"Unable to delete landing zone with ID $landingZoneId for BPM-backed billing project.",
```

Comment:
The resulting error message gets stored on the project, now - so it seemed superfluous to pass in the project name just to tack it on to an error message.